src/screens/SubTags/SubTags.tsx

```
@@ -175,23 +151,23 @@ function SubTags(): JSX.Element {
         </div>
       </div>
     );
   }
 
-  const subTagsList =
-    subTagsData?.getChildTags.childTags.edges.map((edge) => edge.node) ?? [];
-
-  const parentTagName = subTagsData?.getChildTags.name;
+  const parentTag = subTagsData?.getChildTags;
+  const childTagsEdges = parentTag?.childTags?.edges ?? [];
+  const subTagsList = childTagsEdges.map((edge: { node: any }) => edge.node);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)